datasets/SWiFT/temperature_profile_reconstruction.ipynb

```diff
@@ -0,0 +1,1128 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    fig.savefig('figures/combined_radar_met_sounding_thetav.png',bbox_inches='tight')
</pre><p>Are you sure this is correct? Isn't this saving the previous figure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/a2e-mmc/assessment/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>